src/view/dataElements/identityMap.html

```diff
@@ -1,12 +1,12 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <title>Extension View</title>
-</head>
-<body>
-<div id="root"></div>
-<script src="https://assets.adobedtm.com/activation/reactor/extensionbridge/extensionbridge.min.js"></script>
-<script src="identityMap.jsx"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <title>Extension View</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script src="https://assets.adobedtm.com/activation/reactor/extensionbridge/extensionbridge.min.js"></script>
+    <script src="./identityMap.jsx"></script>
+  </body>
 </html>
```

Comment:
These two HTML files were fine the way they were, but I decided to make them consistent with all our other HTML files.